distribution/docs/src/main/resources/_contents/_eventing/eventing-intro-contents.adoc

```diff
@@ -0,0 +1,49 @@
+
+.Eventing Architecture
+[ditaa, catalog_architecture_eventing, png, ${image-width}]
+....
++------------------------------------------------------------+
+|                /-------------------\                       |
+|                |cDEFEndpoints      |                       |
+|                +------------+------+                       |
+|                |cDEF        |cDEF  |                       |
+|                | Operations | Data |                       |
+|/---------------+------------+------+------------+---------\|
+||cDEF           |cDEF               |cDEF        |cDEF     ||
+||  Transformers |                   | Federation | Sources ||
+|+---------------+ Catalog Framework +------------+---------+|
+||cDEF           |                   |c369   Eventing       ||
+||   Catalog     |                   +------------+---------+|
+||   Plugins     |                   |cDEF   Resources      ||
+|\---------------+-------------------+----------------------/|
+|                |cDEF               |                       |
+|                | Catalog Provider  |                       |
+|                +-------------------+                       |
+|                |cDEF               |                       |
+|                | Storage Provider  |                       |
+|                \-------------------/                       |
++------------------------------------------------------------+
+....
+
+The Eventing capability of the Catalog allows endpoints (and thus external users) to create a "standing query" and be notified when a matching metacard is created, updated, or deleted.
+
+Notably, the Catalog allows event evaluation on both the previous value (if available) and new value of a Metacard when an update occurs.
+
+Eventing allows ${branding}s to receive events on operations (e.g. create, update, delete) based on particular queries or actions.
+Once subscribed to an event, users will receive notifications whenever a certain event, such as creation or update of a type of resource, on any sources.
```

Comment:
It may just be me, but this sentence seems a little clunky.  Maybe something simpler?  `Once subscribed, users will receive notifications of events such as update or create on any source.`